### ISSUE TITLE
Git/Github collector

### ISSUE THREAD

**Issue by mlieberman85** [guacsec/guac#205]
It would be nice to have a collector that can clone/pull down files from Git. It would also be useful to have a collector that can ingest from Github releases directly.

**Comment by lumjjb** [comment]
@mlieberman85 can you help specify a bit more on this issue and i'd like to flag it as a good first issue (separate from the git document collector)... maybe just take in as input a list of github repo URLs and then track the release information and create attestations based on that.

@shafeeshafee can you create a separate issue for the git document collector?

**Comment by pdxjohnny** [comment]
Should this issue have been closed by https://github.com/guacsec/guac/pull/498

**Comment by pxp928** [comment]
@pdxjohnny yes this issue can be closed. If you have some comments or recommendations we can open a new issue.
